src/stcal/jump/jump.py

```diff
@@ -734,16 +747,10 @@ def extend_ellipses(
         axis1 = min(axis1, max_extended_radius)
         axis2 = min(axis2, max_extended_radius)
         alpha = ellipse[2]
-        image = cv.ellipse(
-            image,
-            (round(ceny), round(cenx)),
-            (round(axis1 / 2), round(axis2 / 2)),
-            alpha,
-            0,
-            360,
-            (0, 0, jump_flag),
-            -1,
-        )
+        center = (round(ceny), round(cenx))
+        axes = (round(axis1 / 2), round(axis2 / 2))
+        rr, cc = skimage.draw.ellipse(*center, *axes, shape=image.shape, rotation=alpha)
+        image[rr, cc, 2] = 4
```

Comment:
is there a reason why this is rounded to the nearest integer?

Reply:
If I remember right the routine is expecting integers.  My guess was this is just an interface to C code.
Eventually we have to more to integers to mask the pixels anyway.